Makefile

```diff
@@ -36,6 +36,15 @@ include .make/sdk/main.make
 messages:
 	@$(GO) run ./cmd/internal/generate_i18n.go
 
+docs-gen:
+	hugo -s ./doc --baseUrl https://github.com/TheThingsNetwork/lorawan-stack/$(GIT_TAG)/ -d public/$(GIT_TAG)
+
+docs-server:
+	hugo server -s ./doc
+
+doc-deps:
+	@echo https://gohugo.io/getting-started/installing
```

Comment:
Should we perhaps fall back to `go run`ning hugo if it's not installed?

Reply:
Hugo is quite heavy to pull and install, much like the stack.
Instead, I'll add instruction in the DEVELOPMENT.md or the doc README.md on how to install `hugo`

Using `snap` for the travis build.